### PR TITLE
Added display for number of hints.

### PR DESCRIPTION
**Pull request by vokounigor**

### CODE REVIEW

**Comment by alexmojaki** [comment]
Sweet, it's nice to have these previews.

Let's try making this a little cleaner:

1. Don't show the indicator when it's shown all hints
2. That should mean that "hint(s)" can just be "hints".
3. I *think* it'd be nicer to have it directly under the button, in slightly smaller text.

**Comment by vokounigor** [comment]
Previews sure are handy! Check it out now.

File: frontend/src/Hints.js [hunk 1]
```diff
@@ -58,6 +58,10 @@ const Hints = ({hints, numHints, requestingSolution, solution}) =>
           />
       }
     </div>
+    {
+      numHints !== hints.length &&
+      <span className="small">Shown {numHints} of {hints.length} hints.</span>
```
Comment (alexmojaki) [inline frontend/src/Hints.js:63]:
```suggestion
      numHints < hints.length &&
      <span className="small">Shown {numHints} of {hints.length} hints</span>
```

I moved a step and it said "Shown 10 of 5"

**Comment by alexmojaki** [comment]
This is great, thank you! What do you think about having a chat on slack?

**Pull request merged**